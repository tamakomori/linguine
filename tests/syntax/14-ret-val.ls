func main() {
   print("return " + foo());
   print("return " + bar("abc"));
}

func foo() {
   return 123;
}

func bar(a) {
    return a;
}
