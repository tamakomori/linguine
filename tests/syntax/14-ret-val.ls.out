return 123
return abc
